src/Camera.xml

```diff
@@ -0,0 +1,114 @@
+        </property>
+        <property key="width" type="integer" defaultValue="714">
+            <caption>Width</caption>
+            <category>Appearance</category>
+            <description/>
+        </property>
+        <property key="heightUnit" type="enumeration" defaultValue="pixels">
+            <caption>Height unit</caption>
+            <category>Appearance</category>
+            <description>'Percentage of width' is the aspect ratio, 'Pixels' is absolute.</description>
+            <enumerationValues>
+                <enumerationValue key="percentageOfWidth">Percentage of width</enumerationValue>
+                <enumerationValue key="pixels">Pixels</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="height" type="integer" defaultValue="477">
+            <caption>Height</caption>
+            <category>Appearance</category>
+            <description/>
+        </property>
+        <property key="fileType" type="enumeration" defaultValue="jpeg">
+            <caption>File type</caption>
+            <category>Behaviour</category>
+             <description>Select the format in which the image is captured</description>
+            <enumerationValues>
+                <enumerationValue key="webp">webp</enumerationValue>
+                <enumerationValue key="jpeg">jpeg</enumerationValue>
+                <enumerationValue key="png">png</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="imageFilter" type="enumeration" defaultValue="none">
+            <caption>Image filter</caption>
+            <category>Appearance</category>
+             <description>Filter to be applied to the image</description>
+            <enumerationValues>
+                <enumerationValue key="grayscale">Grayscale</enumerationValue>
+                <enumerationValue key="none">Color</enumerationValue>
+                <enumerationValue key="sepia">Sepia</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="photo" type="entity" required="true">
+            <caption>Select entity</caption>
+            <category>Behaviour</category>
+            <description>This is the entity property</description>
+        </property>
+        <property key="captureButtonName" type="translatableString" required="false" >
+            <caption>Capture button</caption>
+            <category>Button captions</category>
+            <description>The class on the button that captures the image. This button should not have an action</description>
+            <translations>
+                <translation lang="en_US">Take photo</translation>
+                <translation lang="nl_NL">Maak enn foto</translation>
+            </translations>
+        </property>
+        <property key="captureButtonIcon" type="string" defaultValue="camera">
+            <caption>Capture icon</caption>
+            <category>Button captions</category>
```

Comment:
This category is too specific for the items you've added to it. Perhaps rename it to something more general... like `Buttons`

Reply:
done